app/task_routes.py

```diff
@@ -0,0 +1,125 @@
+from flask import Blueprint, request, make_response, jsonify, abort
+from sqlalchemy import asc, desc
+from app.models.task import Task
+from app import db
+from datetime import datetime
+import os
+import requests
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def get_validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+
+@tasks_bp.route("", methods=["GET"])
+def read_all_tasks():
+    order_by = request.args.get("sort")
+
+    if order_by == 'asc':
+        tasks = Task.query.order_by(asc('title')).all()
+    elif order_by == 'desc':
+        tasks = Task.query.order_by(desc('title')).all()
+    else:
+        tasks = Task.query.all()
+
+    tasks_list = [task.to_dict() for task in tasks]
+    return make_response(jsonify(tasks_list), 200)
+
+
+@tasks_bp.route("/<task_id>", methods=["GET"])
+def read_one_task(task_id):
+    task = get_validate_model(Task, task_id)
+    current_task_response = {"task": task.to_dict()}
+
+    return make_response(jsonify(current_task_response), 200)
+
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body:
+        return make_response({
+            "details": "Invalid data"
+        }, 400)
+    
+    new_task = Task(title=request_body["title"],
```

Comment:
Line 58 is concerned with creating an instance of the Task class. You could argue that the logic in this method create_task() should only be concerned with creating a valid response to return to the client. Therefore, the logic on line 58 should be moved into the Task class so that the Task class is responsible for creating an instance of itself and we can just invoke the method here. 

If you create a class method from_dict() in the Task class, then line 58 would just be:
```python
new_task = Task.from_dict(request_body) 
```

That way we don't need to expose the logic that belongs in Task in a route